kombu/transport/kafka.py

```diff
@@ -0,0 +1,336 @@
+"""
+kombu.transport.kafka
```

Comment:
I don't know that much about the kafka ecosystem, but do we really want to have more than one kafka transport at the same time?  If kafka_python is better, then could it not replace the pykafka based one later?  I think it'd be easier for users if they don't have multiple choices.

I did the same executive decision for the cassandra result backend in Celery 4.0rc, where the new version was first suggested to be named 'new_cassandra'.

If there's a compelling reason for a small subset of the users to continue using the older version, it's possible to publish a compatibility extension on PyPI.